client/src/elements/schema-editor/schema-editor-table.html

```diff
@@ -4,5 +4,5 @@
     ${schema["title"]}
     <div ref="tableContainerElement"></div>
   </label>
-  <button-secondary if.bind="options.allowTranspose" click.delegate="transpose()" icon="transpose" style="width: 36px;"></button-secondary>
+  <button-secondary if.bind="options.allowTranspose" click.delegate="transpose()">${ 'editor.transpose' & t }</button-secondary>
```

Comment:
Could we keep the icon as well? `button-secondary` should handle that if you just keep the icon attribute.